### PR TITLE
Clean up some data, using macros for multiline list entries

### PR DESCRIPTION
**Pull request by Rangi42**

### CODE REVIEW

**Comment by Rangi42** [comment]
Sorry, I force-pushed a few more edits; I'll wait for re-approval before merging.

This is basically everything I have in mind for the data/ formatting. A major thing left is to define more constants instead of raw numbers; to actually use the constants that are already defined; and to add comments correlating the constants, data, and relevant engine parts.

File: data/sprites/sprites.asm [hunk 1]
```diff
@@ -1,360 +1,80 @@
...
-	db $40 ; byte count
-	db BANK(LyingOldManSprite)
-
-	; SPRITE_LYING_OLD_MAN
-	dw LyingOldManSprite
-	db $40 ; byte count
-	db BANK(LyingOldManSprite)
+SpriteSheetPointerTable:
+; graphics, tile count
+	overworld_sprite RedSprite, 12 ; SPRITE_RED
```
Comment (dannye) [inline data/sprites/sprites.asm:9]: Align these comments perhaps?

Comment (Rangi42) [reply data/sprites/sprites.asm:9]: I don't think there's been any convention for whether this sort of comment should be indented or not, but sure, I can indent them.

Comment (Rangi42) [reply data/sprites/sprites.asm:9]: Actually, a lot of this sort of comment should be revised to have comments linking constants and data together, so I'll do that separately.

**Pull request merged**